Dockerfile

```diff
@@ -317,6 +304,20 @@ USER postgres
 
 ENV MAKEFLAGS=-j4
 
+# pgai is an extension for artificial intelligence workloads
+ARG PGAI_VERSION
```

Comment:
Notice line 303, this is in a `USER postgres` context, instead of previously being in `USER root`